lib/private/TemplateLayout.php

```diff
@@ -108,8 +108,8 @@ public function __construct($renderAs, $appId = '') {
 
 			$this->initialState->provideInitialState('core', 'active-app', $this->navigationManager->getActiveEntry());
 			$this->initialState->provideInitialState('core', 'apps', $this->navigationManager->getAll());
-			
-			if ($this->config->getSystemValueBool('unified_search.enabled', false)) {
+
+			if ($this->config->getSystemValueBool('unified_search.enabled', false) || !$this->config->getSystemValueBool('enable_non-accessible_features', true)) {
```

Comment:
@Fenn-CS I've added the former switch back. However, I have some doubt about how it is used. Can you double-check and confirm that the condition is reversed?

Shouldn't it be:

```suggestion
			if (!$this->config->getSystemValueBool('unified_search.enabled', false) || !$this->config->getSystemValueBool('enable_non-accessible_features', true)) {
```

Reply:
@artonge It looks okay, the confusion is stemming form the name, the new search was called "global search" so, this was to turn on the former search called "unified search" ... However all of them are now called unified search so that seems off...

it could be renamed to `unified_search.enabled`.

All that it does it toggle between showing the new search modal or the former search tooltip!